test/record/record-write-acknowledgementSpec.js

```diff
@@ -0,0 +1,210 @@
+'use strict'
+
+/* global it, describe, expect, jasmine */
+let proxyquire = require('proxyquire'),
```

Comment:
i know the rest of the files do this, but for new files can we use the 

const x = req
const y = req

syntax. I actually found a few bugs in tests due to missing commas